forcing_prep/generate.py

```diff
@@ -23,56 +37,53 @@
 import dask.dataframe as ddf
 
 
-def process_geo_data(gdf, data, name):
+def process_geo_data(gdf, data, name, y_lat_dim, x_lon_dim,out_dir = '', redo = False, cvar = 8, ctime_max = 120, cid = -1):
     print("Slicing data to domain")
     # Only need to load the raster for the geo data extent
     extent = gdf.total_bounds
     lats = slice(extent[1], extent[3])
     lons = slice(extent[0], extent[2])
-    # In  case the data is upside down, flip the y axis
```

Comment:
This check should be left in, I think.  The AORC rasters are typically "upside down" and if it isn't flipped here before selecting the subset, you may not get correct extent.

Reply:
I'll need to find where I was having issues with the flipping. In some cases, it caused one of the coordinates (latitude?) to have a dimension of size 0.

Reply:
This should be fixed in #4 addressing a bug in the original logic.